public/app/plugins/datasource/grafana-azure-monitor-datasource/dashboards/vMInsightsWorkspace.json

```diff
@@ -0,0 +1,915 @@
+{
+  "annotations": {
+    "list": [
+      {
+        "builtIn": 1,
+        "datasource": "-- Grafana --",
+        "enable": true,
+        "hide": true,
+        "iconColor": "rgba(0, 211, 255, 1)",
+        "name": "Annotations & Alerts",
+        "target": {
+          "limit": 100,
+          "matchAny": false,
+          "tags": [],
+          "type": "dashboard"
+        },
+        "type": "dashboard"
+      }
+    ]
+  },
+  "editable": true,
+  "fiscalYearStartMonth": 0,
+  "gnetId": null,
+  "graphTooltip": 0,
+  "id": null,
+  "iteration": 1644448161950,
+  "links": [],
+  "liveNow": false,
+  "panels": [
+    {
+      "collapsed": false,
+      "datasource": null,
+      "gridPos": {
+        "h": 1,
+        "w": 24,
+        "x": 0,
+        "y": 0
+      },
+      "id": 22,
+      "panels": [],
+      "title": "Top Virtual Machines and Scale Sets by Performance Metrics",
```

Comment:
So what does "Top Virtual Machines" mean in this context? It looks like you can select the average and P5th so wouldn't that be like the bottom then? Maybe better to change this to "Virtual Machine Performance Metrics" or something? I also don't know that a title is required if there's just one section, you might be able to remove it entirely?

2 other small suggestions if it's possible:
- is it possible to pre-select an Aggregate?
- what do you think about pre-selecting all resource groups by default?

Reply:
Top here is referring to top by most concerning values, not necessarily highest or lowest. I am working on adding a few more panels in this PR and in that scenario the rows will make more sense! 

Regarding the two suggestions, I like both of them so I will look into incorporating them!

Reply:
I have added the changes I mentioned here. Regarding the two suggesstions: 
- I preselected the aggregate
- Due to performance concerns, we decided against preselecting all resource groups, to make up for this though, I have added an informative panel at the top of the dashboard that points the user to changing the Resource group or Workspace template variable if they are not seeing any data displayed so that they don't think the dashboard is broken. Let me know what you think!